examples/save_data_example.py

```diff
@@ -0,0 +1,92 @@
+from numpy import *
+from pyspecdata import *
+import os
+import sys
+import SpinCore_pp
+import time
+from Instruments import power_control
+from Instruments import Bridge12,prologix_connection,gigatronics
+from Instruments.XEPR_eth import xepr
+from SpinCore_pp.power_helper import gen_powerlist
+#{{{ params for Bridge 12/power
+dB_settings = gen_powerlist(max_power,power_steps)
+powers = 1e-3*10**(dB_settings/10.)
+#}}}
+DNP_data = None
+#{{{ simplified pulse prog
+def run_scans(nScans, power_idx, field_idx, DNP_data=None):
+    """run nScans and slot them into the power_idx index of DNP_data -- assume
+    that the first time this is run, it will be run with DNP_data=None and that
+    after that, you will pass in DNP_data
+    
+    assume that the power axis is 1 longer than the "powers" array
+    (note that powers and other parameters are defined globally w/in the
+    script, as this function is not designed to be moved outside the module
+    """
+    for x in range(nScans):
+        run_scans_time_list = [time.time()]
+        SpinCore_pp.configureTX(26, 14.894179, r_[0.0,90.0,180.0,270.0], 1.0, 2048)#adc_offset, carrierFreq_MHz, tx_phases, amplitude, nPoints
+        acq_time_ms = SpinCore_pp.configureRX(24, 2048, 1, 1, 4)#SW_kHz, nPoints, 1, nEchoes, nPhaseSteps
+        SpinCore_pp.init_ppg()
+        SpinCore_pp.load([
+            ('marker','start',1),
+            ('phase_reset',1),
```

Comment:
so I don't understand why you don't just delete the entire run_scans function and replace it with a delay (where it's called below).  Nothing inside run_scans affects the functioning of your log, and then you have removed the dependence on SpinCore_pp in one swoop.

Reply:
deleted the run_scans function

Reply:
okay replaced with delay